erdpy/config.py

```diff
@@ -162,10 +165,56 @@ def get_defaults() -> Dict[str, Any]:
 
 
 def read_file() -> Dict[str, Any]:
-    if not os.path.isfile(CONFIG_PATH):
-        return dict()
-    return utils.read_json_file(CONFIG_PATH)
+    if os.path.isfile(LOCAL_CONFIG_PATH):
+        return utils.read_json_file(LOCAL_CONFIG_PATH)
+    if os.path.isfile(CONFIG_PATH):
+        return utils.read_json_file(CONFIG_PATH)
+    return dict()
 
 
 def write_file(data: Dict[str, Any]):
-    utils.write_json_file(CONFIG_PATH, data)
+    if os.path.isfile(LOCAL_CONFIG_PATH):
+        utils.write_json_file(LOCAL_CONFIG_PATH, data)
+    else:
+        utils.write_json_file(CONFIG_PATH, data)
+
+
+def add_config_args(argv):
+    try:
+        func, subcommand, *_ = argv
+    except ValueError:
+        return argv
+
+    config = read_file()
```

Comment:
Consider using `resolve_config_path()` and passing it as argument directly to `utils.read_json_file()` here, obviating the need for the function `read_file()` above.

Reply:
This `read_file()` function is used in multiple places and it's also used outside of the `utils` module. I'm keeping this as-is.